website/docs/getting-started.md

```diff
@@ -8,9 +8,9 @@ import { getCapacitorVersion, getPortalsVersion, getPortalsVersionIos, getPortal
 
 ## Using your Product Key
 
-To use Ionic Portals, you need to register for a product key.
+To use Ionic Portals, you need to a product key. If you don't have a product key please [contact sales](https://ionic.io/demo?source=portals) to run a pilot.
```

Comment:
I imagine this was supposed to say “you need to have a product key”

Reply:
Good catch, fixed.